### ISSUE TITLE
Docs: Add step-by-step navigation to complete the setup of UCP

### ISSUE THREAD

**Issue by joaofnfernandes** [docker/docker.github.io#1687]
@mbentley commented on

### What's the problem/feature request?
Users would prefer if the documentation for UCP would include more of a step-by-step guide for what do.  This could be a page with links to the different documentation sections.  This should help with being able to complete a POC.  This would act as a walkthrough/guide to complete the installation and ensure that the value of the product is seen.

They found the documentation to be a bit all over; having to jump between sections.

### What versions/components are affected?
UCP 2.x

---

@joaofnfernandes commented 

Does the [DDC for AWS](https://docs.docker.com/datacenter/install/aws/) and [DDC for Linux](https://docs.docker.com/datacenter/install/linux/) docs help or do we need a bit more guidance?

(Don't be afraid to say that we need a lot more guidance. I've see how participants in our training session can struggle installing DDC)

---

@mbentley commented on

Unfortunately, no.  This would be more of a guide walking through the installation options, presenting the different choices to the user, for example:

- Install UCP
  - Single controller
  - HA
- Adding nodes
- Configuring TLS
- Configuring auth
  - AD/LDAP
  - Managed by UCP
- Creating Teams
  - AD/LDAP
  - Managed by UCP

Basically this would be something that steps a user though the complete setup of UCP.  There are different scenarios that a user needs to consider and all of the fine grained details.  When going through the install process with Walgreens, we found ourselves jumping around in the documentation and they found it confusing and somewhat frustrating.  Links directly to the sections from each bullet point would be sufficient for the "guide" to function as a way to make sure 1.) you're doing everything correct, and 2.) you're using the product fully as intended.

When working with users on installing UCP and DTR, I find myself being the guide to tell people where to go next, the docs leave it a bit open ended without any sort of checklist.

**Comment by johndmulhausen** [comment]
Issue moved to [docker/docs-private #446](https://github.com/docker/docs-private/issues/446) via [**ZenHub**](https://www.zenhub.com/)
